CMakeLists.txt

```diff
@@ -153,3 +154,16 @@ if(ENABLE_TESTS)
 endif()
 
 add_subdirectory(src)
+
+# After all compiled, look for the respective flags
+if(NOT OPENROAD_GPU_COMPILED)
```

Comment:
Why is this condition needed?

Reply:
I wrote it to be consistent with the same syntax as the above commands. 
![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/39641663/52eedac4-d2bd-4470-931e-bf8f57373161)